src/main/java/com/minecolonies/blockout/controls/ButtonHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.minecolonies.blockout.controls;
+
+import java.util.function.Consumer;
+
+/**
+ * Used for windows that have buttons and want to respond to clicks.
+ */
+@FunctionalInterface
+public interface ButtonHandler extends Consumer<Button>
+{
+    /**
+     * Called when a button is clicked.
+     *
+     * @param button the button that was clicked.
+     */
+    void onButtonClicked(Button button);
+
+    default void accept(Button button) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)